### PR TITLE
Various modifications to the scan server ITs

### PR DESCRIPTION
**Pull request by DomGarguilo**
- Created variables with descriptive names to replace hardcoded numbers
- Refactored common code into methods
- Other misc. changes

### CODE REVIEW

**Review by dlmarion — COMMENTED** [review]
Overall looks good.

File: test/src/main/java/org/apache/accumulo/test/ScanServerIT.java [hunk 1]
```diff
@@ -240,4 +197,18 @@ public void testBatchScannerTimeout() throws Exception {
       }
     }
   }
+
+  protected static void createTableAndIngest(AccumuloClient client, String tableName)
+      throws Exception {
+    createTableAndIngest(client, tableName, new NewTableConfiguration());
+  }
+
+  protected static void createTableAndIngest(AccumuloClient client, String tableName,
```
[the remark this answers is not in the document]
Comment (DomGarguilo) [reply test/src/main/java/org/apache/accumulo/test/ScanServerIT.java]: All suggestions should be addressed in 90aaaa1

File: test/src/main/java/org/apache/accumulo/test/ScanServerIT.java [hunk 2]
```diff
@@ -132,46 +127,20 @@ public void testBatchScan() throws Exception {
...
+        assertEquals(EXPECTED_INGEST_ENTRIES_COUNT, Iterables.size(scanner));
         scanner.setConsistencyLevel(ConsistencyLevel.IMMEDIATE);
-        assertEquals(200, Iterables.size(scanner));
+        assertEquals(EXPECTED_INGEST_ENTRIES_COUNT * 2, Iterables.size(scanner));
       } // when the scanner is closed, all open sessions should be closed
     }
   }
 
-  @Test
-  public void testScanOfflineTable() throws Exception {
```
[the remark this answers is not in the document]
Comment (DomGarguilo) [reply test/src/main/java/org/apache/accumulo/test/ScanServerIT.java]: Addressed in cfa97a1

File: test/src/main/java/org/apache/accumulo/test/ScanServerIT_NoServers.java [hunk 3]
```diff
@@ -106,43 +102,17 @@ public void testBatchScan() throws Exception {
...
-        // since there are no scan servers and we are reading from tservers we should see update
-        assertEquals(200, Iterables.size(scanner));
+        // since there are no scan servers, and we are reading from tservers, we should see update
+        assertEquals(EXPECTED_INGEST_ENTRIES_COUNT * 2, Iterables.size(scanner));
       } // when the scanner is closed, all open sessions should be closed
     }
   }
 
-  @Test
-  public void testScanOfflineTable() throws Exception {
```
[the remark this answers is not in the document]
Comment (DomGarguilo) [reply test/src/main/java/org/apache/accumulo/test/ScanServerIT_NoServers.java]: Addressed in cfa97a1

**Review by dlmarion — APPROVED** [review]

**Pull request merged**